clusterloader2/testing/load/config.yaml

```diff
@@ -130,6 +138,16 @@ steps:
       - basename: daemonset-priorityclass
         objectTemplatePath: daemonset-priorityclass.yaml
 
+- module:
```

Comment:
Can we add a TODO (and open an issue) to merge back those sections once the bug in upstream will be fixed?

Reply:
Added

Reply:
nit: "remove when k/k bug is fixed"?

Reply:
Fixed PTAL